deploy/application/environments/env_name.tfvars.jinja

```diff
@@ -0,0 +1,4 @@
+environment = @{{ env_name }}
+app_subdomain = @{{ app_subdomain }}
+tf_state_bucket = @{{ bucket_name }}
```

Comment:
not sure we use this ? could we not use the ssm store and remove the remote state ?

Will need to change in main.tf the remote state stufff with something like:
```
module "ssm_lookup" {
  source = "../ssm" # sub-module of our kubernetes-platform module

  base_prefix       = "infrastructure"
  stack_type        = "platform"
  stack_name_prefix = "discovery" # EKS name without the random suffix

  lookup = [
    "cluster_endpoint",
    "cluster_name"
  ]

  tags = {
    Environment = "sandbox"
    Terraform   = "true"
  }
}

locals {
  stack = module.ssm_lookup.stacks[0]
}

data "aws_eks_cluster" "cluster" {
  name = module.ssm_lookup.lookup[local.stack].cluster_name
}

provider "kubectl" {
  apply_retry_count      = 5
  host                   = data.aws_eks_cluster.cluster.endpoint
  cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority[0].data)
  load_config_file       = false

  exec {
    api_version = "client.authentication.k8s.io/v1beta1"
    command     = "aws"
    args        = ["eks", "get-token", "--cluster-name", data.aws_eks_cluster.cluster.id]
  }
```

Reply:
@rbapst-tamedia We could yes :) 

I think it's useful at the moment to use remote state first and when we have all clusters migrated to the platform. I'd leave it like that for now though if you agree.